fission-cli/library/Fission/CLI/Handler/Generate.hs

```diff
@@ -0,0 +1,29 @@
+module Fission.CLI.Handler.Generate (interpret) where
+
+import           Fission.Prelude
+
+import qualified Fission.CLI.Base.Types                    as Base
+
+import           Fission.CLI.Types
+
+import qualified Fission.CLI.Handler                       as Handler
+
+import           Fission.CLI.Parser.Command.Generate.Types as Generate
+
+import           Fission.CLI.Handler.Error.Types           (Errs)
+
+interpret ::
+  ( Contains Errs errs
+  , Contains errs errs
+  , Display   (OpenUnion errs)
+  , Exception (OpenUnion errs)
+  )
+  => Generate.Options
+  -> FissionCLI errs Base.Config ()
+interpret cmd = do
+  logDebug @Text "Generate interpreter"
+
+  case cmd of
+    Credentials _ -> do
+      Handler.credentials
+      return ()
```

Comment:
No need for this `return`!

Reply:
(or the `do`)

Reply:
Sorry, I wanted to get those comments in before it got merged. `Handler.credentials :: m ()`, so no need for the extra `return () :: m ()`

Reply:
Thanks! Removed it. 👌

Reply:
And removed the `do` as well